src/main/java/net/dkamps18/channelpoints/Handler/CommandHandler.java

```diff
@@ -84,7 +82,7 @@ public boolean run(CommandSender sender, Command cmd, String label, String[] arg
 						}
 					}
 				} else {
-					p.spigot().sendMessage(this.authmsg);
+					p.sendMessage(this.authmsg);
```

Comment:
You can't send a Text component with the normal api

Reply:
Doable with PaperSpigot, I just assumed that was the usual but I can revert this.